publiccode.yml

```diff
@@ -0,0 +1,108 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+name: OpenProject
+applicationSuite: openDesk
+url: 'https://github.com/opf/openproject'
+roadmap: 'https://www.openproject.org/roadmap'
+releaseDate: '2025-01-21'
+softwareVersion: '15.1.1'
+developmentStatus: stable
```

Comment:
Corrected from `development`. This should point to the last public release, not any pre-releases